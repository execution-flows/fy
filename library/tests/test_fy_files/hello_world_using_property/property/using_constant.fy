property greeting using constant:

    @cached
    def -> str:
        return "Hello world!"
