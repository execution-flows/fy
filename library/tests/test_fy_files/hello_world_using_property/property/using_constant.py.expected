from functools import cached_property


class Greeting_UsingConstant_PropertyMixin:
    @cached_property
    def _greeting(self) -> str:
        return "Hello world!"
